### PR TITLE
remove feedback capability (issue #65)

### PR DESCRIPTION
**Pull request by benjaminp**

### CODE REVIEW

File: static/sass/mq.css [hunk 1]
```diff
@@ -525,7 +525,7 @@ html[xmlns] .slides { display: block; }
       /* We do this more explictly (the descendant selector) to avoid also styling links in the Supernav content */ }
       .main-navigation .tier-1 > a, .main-navigation .tier-2 > a {
         display: block;
-        padding: .5em 1.5em .4em 1em;
+        padding: 0.5em 1.5em 0.4em 1em;
```
Comment (birkenfeld) [inline static/sass/mq.css:528]: these changes appear unrelated?

**Comment by birkenfeld** [comment]
Does this leave the small button on the right with instructions about this tracker?

File: static/sass/mq.css (hunk 1 above)
Comment (benjaminp) [inline static/sass/mq.css:528]: Every css file is generated from a .scss file.

**Comment by benjaminp** [comment]
Nope.

**Comment by birkenfeld** [comment]
Do we _want_ to leave it there? :)

File: static/sass/mq.css (hunk 1 above)
Comment (birkenfeld) [inline static/sass/mq.css:528]: So you have a different sass version that generates slightly different CSS?

Comment (benjaminp) [inline static/sass/mq.css:528]: Apparently.

**Comment by benjaminp** [comment]
Note that the current doodad only serves to open the feedback form on the bottom of the page, so we'd have to make some changes to it to provide "instructions about this tracker". IMO, we should just link to the tracker from the bottom of every page of from the /about/help page.

**Comment by birkenfeld** [comment]
Agreed.
